jorgePrecich/ruby/rb_terminal.rb

```diff
@@ -0,0 +1,19 @@
+if ARGV.length < 2
+  puts "You need to write at least 2 values -> ClassName, Attribute"
+else
+  class_name = ARGV.shift().capitalize
+  attributes = ARGV.map! { |x| x.to_sym }
```

Comment:
Same for this, before converting to symbol, it would be good `snakecase`ing it.

Reply:
Yeah, thought the same, I will do it 1st